Description: Rewrite build system as autoconf
 xmkmf is obsolete, so Helmut helpfully rewrote the build system to
 use autoconf instead.
Author: Helmut Grohne <helmut@subdivi.de>
Bug-Debian: https://bugs.debian.org/873807
Last-Update: 2017-09-01

Index: ssh-askpass-1.2.4.1/Makefile.am
===================================================================
--- /dev/null
+++ ssh-askpass-1.2.4.1/Makefile.am
@@ -0,0 +1,16 @@
+libexec_PROGRAMS = x11-ssh-askpass
+x11_ssh_askpass_SOURCES = drawing.c dynlist.c resources.c x11-ssh-askpass.c
+nodist_x11_ssh_askpass_SOURCES = SshAskpass_ad.h
+x11_ssh_askpass_CFLAGS = $(X11_CFLAGS) $(XT_CFLAGS) $(SM_CFLAGS) $(ICE_CFLAGS)
+x11_ssh_askpass_LDADD = $(X11_LIBS) $(XT_LIBS) $(SM_LIBS) $(ICE_LIBS)
+askpassconfdir = $(sysconfdir)/X11/app-defaults
+askpassconf_DATA = SshAskpass
+CLEANFILES = SshAskpass_ad.h x11-ssh-askpass.1x SshAskpass
+man1_MANS = x11-ssh-askpass.1x
+BUILT_SOURCES = SshAskpass_ad.h x11-ssh-askpass.1x
+SshAskpass_ad.h:SshAskpass-default.ad
+	sed -n '/^[^!]/s/.*/"&",/p' <$< >$@
+x11-ssh-askpass.1x:x11-ssh-askpass.man
+	cp $< $@
+SshAskpass:SshAskpass-default.ad
+	cp $< $@
Index: ssh-askpass-1.2.4.1/configure.ac
===================================================================
--- /dev/null
+++ ssh-askpass-1.2.4.1/configure.ac
@@ -0,0 +1,10 @@
+AC_INIT([ssh-askpass],[1.2.4.1],[jmknoble@pobox.com])
+AM_INIT_AUTOMAKE([foreign])
+AC_PROG_CC
+PKG_CHECK_MODULES([X11],[x11])
+PKG_CHECK_MODULES([XT],[xt])
+PKG_CHECK_MODULES([SM],[sm])
+PKG_CHECK_MODULES([ICE],[ice])
+AC_SUBST(NAME,[x11-ssh-askpass])
+AC_CONFIG_FILES([Makefile x11-ssh-askpass.man])
+AC_OUTPUT()
Index: ssh-askpass-1.2.4.1/x11-ssh-askpass.man.in
===================================================================
--- ssh-askpass-1.2.4.1.orig/x11-ssh-askpass.man.in
+++ ssh-askpass-1.2.4.1/x11-ssh-askpass.man.in
@@ -3,7 +3,7 @@
 .\" Modified by Jim Knoble <jmknoble@pobox.com> for non-OpenBSD
 .\"   distribution
 .\"
-.Dd @DATE@
+.Dd September 17, 2001
 .Dt @NAME@ 1 
 .Os "Version @VERSION@"
 .Sh NAME
